### ISSUE TITLE
[BUG]  UI elements not shown properly Win 11

### ISSUE THREAD

**Issue by gt2416** [snappysnappydog/Simple-IP-Config#204]
**Version:**
2.9.7

**Expected Behavior:**
The UI elements and text look normal like the screenshots from the github main page.

**Actual Behavior:**
Running on Windows 11 23H2 the UI does not look proper.

![image](https://github.com/KurtisLiggett/Simple-IP-Config/assets/34902894/366319e1-b1a2-4e5a-a7c6-da26ef27d32e)

**Comment by fongwithroot** [comment]
Yeah I get the same, plus sometimes the APPLY button appears only one or two pixels high. Just barely enough to click.

![Image](https://github.com/user-attachments/assets/48a502d8-df23-4f78-8253-4e6989006cee)
